build.cake

```diff
@@ -267,6 +269,19 @@ Task("TestNet50EngineCore")
         RunDotnetNUnitLiteTests(NETCORE_ENGINE_CORE_TESTS, "net5.0");
     });
 
+//////////////////////////////////////////////////////////////////////
+// TEST NET 5.0 ENGINE CORE
+//////////////////////////////////////////////////////////////////////
+
+Task("TestNet60EngineCore")
+    .Description("Tests the .NET 6.0 Engine core assembly")
+    .IsDependentOn("Build")
+    .OnError(exception => { UnreportedErrors.Add(exception.Message); })
+    .Does(() =>
+    {
+        RunDotnetNUnitLiteTests(NETCORE_ENGINE_CORE_TESTS, "net6.0");
+    });
+
 //////////////////////////////////////////////////////////////////////
 // TEST .NET 2.0 ENGINE
 //////////////////////////////////////////////////////////////////////
```

Comment:
I cannot comment on the line, but I was going to ask why we didn't add "net6.0" in `TestNetCore31Engine`, but now I can see that we actually don't use `runtime` in the loop, so we are just running `RunDotnetNUnitLiteTests(NETCORE_ENGINE_TESTS, "netcoreapp3.1");` two times.

```
        var runtimes = new[] { "netcoreapp3.1", "net5.0" };

        foreach (var runtime in runtimes)
        {
            RunDotnetNUnitLiteTests(NETCORE_ENGINE_TESTS, "netcoreapp3.1");
        }
    });
```

Reply:
Also odd. I'll check this after I make the other changes.

Reply:
Yes... the second intention was to run the engine tests twice, under each runtime. I'll think about whether that's really necessary now.

Reply:
I decided this is no longer necessary, since we actually have .NET 5.0 and 6 .0 builds, which we test separately.